dask/dataframe/core.py

```diff
@@ -579,6 +579,17 @@ def _aca_agg(self, token, func, aggfunc=None, **kwargs):
         """ Wrapper for aggregations """
         raise NotImplementedError
 
+    def apply_concat_apply(self, func, aggfunc=None, func_kwargs=None,
```

Comment:
I prefer `aggfunc` rather than `aggregate` because it gives a clear pointer of that a function is expected there. This also keeps consistency with `_aca_agg`.